### PR TITLE
Fixes multiple-select in darkmode

### PR DESCRIPTION
**Pull request by Asizon**
Fixes muiltiple-select hover in darkmode. This plugin is used for new Presets feature.

Before:
nok

With this PR:
ok

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

File: src/css/dark-theme.css
Comment (limonspb) [inline src/css/dark-theme.css:912]: i think changes in this file are not needed anymore?

File: src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css
Comment (limonspb) [inline src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:76]: could you please add something like:
```
#presets_detailed_dialog
.ms-drop ul>li.hide-radio:focus, .ms-drop ul>li.hide-radio:hover {
```

so that this style has been applied only for this particular dialog.

**Review by limonspb — CHANGES_REQUESTED** [review]

File: src/css/dark-theme.css
Comment (Asizon) [reply src/css/dark-theme.css:912]: Yes we need that fix in multiole-select for future uses.

File: src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css
Comment (Asizon) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:76]: Umm, really needed? Its already on presets_detailed_dialog css file so only works here.

Comment (limonspb) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:76]: this css file is included from main html, so all the styles from this file will be applied to the whole Configurator. Unless you narrow them with more specific selectors.

Comment (Asizon) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.css:76]: Ouch! Understood, thought that it is included amd comes only in presets dialog html and js files.

**Comment by Asizon** [comment]
Done!Ready for merge

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
